config.rb

```diff
@@ -65,6 +65,12 @@ def tel_to(text, className)
 sprockets.append_path File.join(root, "node_modules/govuk-frontend/")
 sprockets.append_path File.join(root, "node_modules/gaap-analytics/build")
 
+# Special handling for redirecting to security.txt as it is a non-html content type.
+# As per https://www.jvt.me/posts/2021/10/28/middleman-redirect-content-type/
```

Comment:
Not sure if we need this link here?

Reply:
I felt it is the most succinct way to explain to the rationale for having two redirects, which is otherwise quite counterintuitive.

Reply:
Would be good to add relevant info to the commit message.

URL may not work after a while and we will lose the context.

Reply:
I think it's too complex to explain in full in a commit message.  As a compromise, I've written a short summary and put the link in the commit message.